test/causal-cluster-compose.yml

```diff
@@ -12,8 +12,11 @@ services:
       - lan
     environment:
       - NEO4J_ACCEPT_LICENSE_AGREEMENT=yes
-      - NEO4J_AUTH=none
+      - NEO4J_AUTH=neo4j/neo
```

Comment:
I think this causes the test-sets-password test to fail

Reply:
Don't change this value

Reply:
ack

Reply:
I'll drop this commit